tests/common.py

```diff
@@ -19,19 +19,27 @@
 # fmt: off
 #                1      2     3    4      5      6      7     8      9    10    11    12
 MNEMONIC12 = "alcohol woman abuse must during monitor noble actual mixed trade anger aisle"
-# fmt: on
-
-MNEMONIC_SHAMIR_20_3of6 = [
+MNEMONIC18 = "owner little vague addict embark decide pink prosper true fork panda embody mixture exchange choose canoe electric jewel"
```

Comment:
ISTM you reintroduced MNEMONIC18, 24 and ALLALLALL, which don't exist in master anymore?